### PR TITLE
refs qorelanguage/qore#2542 fixed a bug where DBI-SELECT-ROW-ERROR ex…

### PR DESCRIPTION
**Pull request by davidnich**
…ceptions were being raised as MYSQL-SELECT-ROW-ERROR exceptions

### CODE REVIEW

File: docs/mainpage.dox.tmpl [hunk 1]
```diff
@@ -166,6 +166,7 @@ hash: (1 member)
     @subsection mysql2021 mysql Driver Version 2.0.2.1
     <b>New Features and Bug Fixes</b>
     - updated SQLStatement::fetchColumns() to return an empty hash when no data is available (<a href="https://github.com/qorelanguage/qore/issues/1241">issue 1241</a>)
+    - fixed a bug where \c DBI-SELECT-ROW-ERROR exceptions were being raised as \c ORACLE-SELECT-ROW-ERROR exceptions (<a href="https://github.com/qorelanguage/qore/issues/2542">issue 2542</a>)
```
Comment (tehe) [inline docs/mainpage.dox.tmpl]: Should've said "as \c MYSQL-SELECT-ROW-ERROR".

**Comment by davidnich** [comment]
@tehe fixed

**Comment by tehe** [comment]
I'm sorry, I don't see the fix.

**Comment by davidnich** [comment]
sorry, it's there now

**Comment by tehe** [comment]
LGTM

**Pull request merged**